ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -299,12 +303,35 @@ public enum CocoaPodUtils {
     target 'FrameworkMaker' do\n
     """
 
+    var versionsSpecified = false
+
     // Loop through the subspecs passed in and use the actual Pod name.
     for pod in pods {
-      let version = pod.version == nil ? "" : ", '\(pod.version!)'"
-      podfile += "  pod '\(pod.name)'" + version + "\n"
+      podfile += "  pod '\(pod.name)'"
+      // Check if we want to use a local version of the podspec.
+      if let localURL = LaunchArgs.shared.localPodspecPath,
+        FileManager.default.fileExists(atPath: localURL.appendingPathComponent(pod.name + ".podspec").path) {
+        podfile += ", :path => '" + localURL.path + "'"
+      } else if pod.version != nil {
```

Comment:
We can use `if let` instead of force unwrapping here:
```
else if let podVersion = pod.version {
  podfile += ", '\(podVersion)'"
}
```

Reply:
Not sure if this was missed or it didn't work, just wanted to mention it incase the comment was lost :) Non blocking for merge though.